src/js/popup.js

```diff
@@ -92,15 +92,19 @@ async function showRelayPanel(tipPanelToShow) {
   //Premium Panel
   const premiumPanelWrapper = document.querySelector(".premium-wrapper");
   const registerDomainImgEl = premiumPanelWrapper.querySelector(".email-domain-illustration");
-
   const aliasesUsedValEl = premiumPanelWrapper.querySelector(".aliases-used");
   const emailsBlockedValEl = premiumPanelWrapper.querySelector(".emails-blocked");
   const emailsForwardedValEl = premiumPanelWrapper.querySelector(".emails-forwarded");
 
+  //Check if premium features are available
+  const premiumEnabled = await browser.storage.local.get("premiumEnabled");
+  const premiumEnabledString = premiumEnabled.premiumEnabled;
```

Comment:
I _think_ this is the only place where we used `premiumEnabled`. Not sure if you can just destructure this and reduce by 1 line.

```js
const { premiumEnabled } = await browser.storage.local.get("premiumEnabled");
```

Reply:
Or you could even wrap this in some helper function which also converts from "True" string to a Boolean `true`.

Reply:
I remember @groovecoder and I tried to destructure lines L100 and L101 but for some reason it would output `undefined` when we tried to console log it.